src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -348,17 +376,18 @@ private MinigameTask cloneBugfinder(final MinigameTask minigameTask, final Strin
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("bugfinder-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("bugfinder-backend not present");
-                    return new MinigameTask(Minigame.BUGFINDER, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```

Comment:
In an exception, debug level? Also the backend name should be mentioned in the log.

Reply:
this is changed in the refactored cloning so i won't change it here